app/javascript/app/providers/ndcs-documents-meta-provider/ndcs-documents-meta-provider-reducers.js

```diff
@@ -3,20 +3,22 @@ import groupBy from 'lodash/groupBy';
 export const initialState = {
   loading: false,
   loaded: false,
-  data: {}
+  data: {},
+  error: false
```

Comment:
As this is a meta provider, we aren't going to fetch per country. So it seems pointless to have an error param on the state. If there is an error set the data to {}. In the front end we are only going to show "no content" at this point. So this serves no purpose.